dc-model/src/main/java/de/digitalcollections/model/filter/FilterCriterion.java

```diff
@@ -21,35 +24,41 @@
  */
 public class FilterCriterion<T extends Object> {
 
-  private String fieldName;
+  private String expression;
   private Comparable<?> maxValue;
-
   private Comparable<?> minValue;
+  private boolean nativeExpression;
   private FilterOperation operation;
   private Object value;
-
   private Collection<?> values;
 
   /**
-   * Constructor for Filter Criteria.
+   * Constructor for a filter criterion.
    *
-   * @param fieldName target field the criteria should be used for
-   * @param operation operation of criteria
-   * @param value operand of criteria
+   * @param expression expression the criterion should be used for
+   * @param nativeExpression a flag to indicate if expression has to be nativeExpression or should
```

Comment:
```suggestion
   * @param nativeExpression a flag to indicate if expression has to be mapped or should
```
I guess it was changed by the IDE when renaming the field